numbers_letters_and_variables/number_of_seconds.rb

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+seconds_in_an_minute = 60
```

Comment:
Usually want a blank line between the UBER line and the first line of the program...